spanner/api/Spanner.Samples.Tests/UpdateUsingDmlWithTimestampCoreAsyncTest.cs

```diff
@@ -28,8 +28,15 @@ public UpdateUsingDmlWithTimestampCoreAsyncTest(SpannerFixture spannerFixture)
     [Fact]
     public async Task TestUpdateUsingDmlWithTimestampCoreAsync()
     {
-        UpdateUsingDmlWithTimestampCoreAsyncSample sample = new UpdateUsingDmlWithTimestampCoreAsyncSample();
-        var rowCount = await sample.UpdateUsingDmlWithTimestampCoreAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.DatabaseId);
-        Assert.Equal(3, rowCount);
+        await _spannerFixture.RunWithTemporaryDatabaseAsync(async databaseId =>
+        {
+            await _spannerFixture.InitializeTempDatabaseAsync(databaseId);
+            AddCommitTimestampAsyncSample addCommitTimestampAsyncSample = new AddCommitTimestampAsyncSample();
+            await addCommitTimestampAsyncSample.AddCommitTimestampAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, databaseId);
+
+            UpdateUsingDmlWithTimestampCoreAsyncSample sample = new UpdateUsingDmlWithTimestampCoreAsyncSample();
+            var rowCount = await sample.UpdateUsingDmlWithTimestampCoreAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, databaseId);
+            Assert.Equal(2, rowCount);
```

Comment:
Why did the assert changed here?

Reply:
Earlier UpdateUsingBatchDmlCoreAsync.cs used to update the Albums tables and insert a new row `"INSERT INTO Albums (SingerId, AlbumId, AlbumTitle, MarketingBudget) VALUES (1, 3, 'Test Album Title', 10000)"` which makes the count to 3. Now as they are independent its just 2 the one which is inserted by init function.